include/grpc/impl/codegen/grpc_types.h

```diff
@@ -368,6 +368,10 @@ typedef struct {
    balancer before using fallback backend addresses from the resolver.
    If 0, enter fallback mode immediately. Default value is 10000. */
 #define GRPC_ARG_GRPCLB_FALLBACK_TIMEOUT_MS "grpc.grpclb_fallback_timeout_ms"
+/* The set of channel args to be passed to the channel established to the grpclb
```

Comment:
This comment is a little misleading in that it implies that either exactly this set of channel args or exactly the set from the parent channel will be used, when in reality the grpclb LB policy will make some changes to the channel args either way.  I suggest the following wording instead:

"""
Channel args to be used for the control-plane channel created to the grpclb load balancers.  This is a pointer arg whose value is a grpc_channel_args object.  If unset, most channel args from the parent channel will be propagated to the grpclb channel.
"""

Reply:
Fixed.